src/components/PlatformNavigator/index.tsx

```diff
@@ -0,0 +1,88 @@
+import { useState, useRef, useEffect } from 'react';
+import { Button, Flex, Text, View, useTheme, VisuallyHidden } from '@aws-amplify/ui-react';
+import { IconChevron } from '@/components/Icons';
+import { frameworks } from '@/constants/frameworks';
+import { InfoPopover } from './InfoPopover';
+import Link from 'next/link';
+import classNames from 'classnames';
+import { useClickOutside } from '@/utils/useClickOutside';
+
+export function PlatformNavigator({ currentPlatform }) {
+  const [isOpen, setIsOpen] = useState<boolean>(false);
+
+  const { tokens } = useTheme();
```

Comment:
Do we need this? We should be able to use `fontWeight="bold"` or `fontWeight="700"`